src/dialogs/examples/basic/O365ConnectorCardActionsDialogV1.ts

```diff
@@ -0,0 +1,206 @@
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Compose Extension").value("Compose Extension"),
+                                    ]),
+                                new teams.O365ConnectorCardMultichoiceInput(session)
+                                    .id("list-3")
+                                    .title(Strings.pick_a_time)
+                                    .isMultiSelect(false)
+                                    .style("compact")
+                                    .choices([
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Bot").value("Bot"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Tab").value("Tab"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Connector").value("Connector"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Compose Extension").value("Compose Extension"),
+                                    ]),
+                                new teams.O365ConnectorCardMultichoiceInput(session)
+                                    .id("list-4")
+                                    .title(Strings.pick_a_time)
+                                    .isMultiSelect(false)
+                                    .style("compact")
+                                    .choices([
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Bot").value("Bot"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Tab").value("Tab"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Connector").value("Connector"),
+                                        new teams.O365ConnectorCardMultichoiceInputChoice(session).display("Compose Extension").value("Compose Extension"),
+                                    ]),
+                            ])
+                            .actions([
+                                new teams.O365ConnectorCardHttpPOST(session)
+                                    .id("cardAction-1-btn-1")
+                                    .name(Strings.send)
+                                    .body(JSON.stringify({
+                                        list1: "{{list-1.value}}",
+                                        list2: "{{list-2.value}}",
+                                        list3: "{{list-3.value}}",
+                                        list4: "{{list-4.value}}",
+                                    })),
+                            ]);
+
+        let section1    = new teams.O365ConnectorCardSection(session)
```

Comment:
Nit: remove extra space (does tslint not catch this?)

Reply:
removed extra space, no tslint error.